src/utils/arrays.ts

```diff
@@ -184,6 +184,8 @@ export function arrayHasDiff(a: any[], b: any[]): boolean {
     }
 }
 
+export type Diff<T> = { added: T[], removed: T[] };
```

Comment:
changing the return type of `arrayDiff` would be great 😇

Reply:
I must've been half asleep when doing this ;(